package.json

```diff
@@ -13,7 +13,7 @@
     "build": "fable-splitter -c splitter.config.js",
     "watch": "fable-splitter -c splitter.config.js -w",
     "test": "fable-splitter -c splitter.test.config.js",
-    "postinstall": "cross-env npm_package_engines_vscode=1.8.1 node ./node_modules/vscode/bin/install"
+    "postinstall": "node ./node_modules/vscode/bin/install"

```

Comment:
Hmm, I think this may be run after installation for [all the consumers of this package](https://docs.npmjs.com/misc/scripts). Could this be run somewhere else (in the build script, for example)?

Reply:
If this is for testing, maybe you can put it in `pretest`